src/components/MenuGroup/MenuGroup.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {BrowserRouter as Router} from 'react-router-dom';
+import {render, screen, fireEvent} from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import MenuGroup from './index';
+
+describe('MenuGroup component', () => {
+  test('renders without crashing', async () => {
+    render(
+      <Router>
+        <MenuGroup title="Test" urlBase="Test" />
```

Comment:
No passing props like this. Create an object before `describe` and use that. Just like I pointed out in the last PR.

Check this commit:
[Tabs tests](https://github.com/thenewboston-developers/Website/commit/5a9442bb197d32a455d35dac5b0ad61beb3a0f78)

Reply:
https://www.thenewboston.com/style-guide/react#prop-spreading

Reply:
It's allowed while writing tests. 😄 

Thanks for bringing this up. I'll update the docs.

Reply:
Ok no worries, just wanted to raise it before I edit it after dinner tonight.